Docs/Luadoc/LuaDocumentation.xml

```diff
@@ -1090,13 +1090,30 @@ save yourself some time, copy this for undocumented things:
 		Set the Actor's effect magnitude in each direction to the given values.
 	</Function>
 	<Function name='effectoffset' return='void' arguments='float fTime'>
-		Set the Actor's effect offset to <code>fTime</code>.
+		Set the Actor's effect offset to <code>fTime</code>.  The offset is added to the time into the effect before calculating percent_through_effect.
 	</Function>
 	<Function name='effectperiod' return='void' arguments='float fTime'>
 		Set the Actor's effect period to <code>fTime</code>.
 	</Function>
-	<Function name='effecttiming' return='void' arguments='float fRampUp, float fAtHalf, float fRampDown, float fAtZero'>
-		Set the Actor's effect timing.
+	<Function name='effecttiming' return='void' arguments='float ramp_to_half, float hold_at_half, float ramp_to_full, float hold_at_zero, float hold_at_full'>

```

Comment:
I think it would be more messy and awkward to make a separate commit with just the documentation change, so I wrote new documentation when I saw how bad it was.